src/libraries/System.Private.CoreLib/src/System/DateOnly.cs

```diff
@@ -822,5 +830,31 @@ public bool TryFormat(Span<char> destination, out int charsWritten, ReadOnlySpan
 
             return DateTimeFormat.TryFormat(GetEquivalentDateTime(), destination, out charsWritten, format, provider);
         }
+
+#if FEATURE_GENERIC_MATH
+        //
+        // IParseable
+        //
+
+        [RequiresPreviewFeatures]
+        static DateOnly IParseable<DateOnly>.Parse(string s, IFormatProvider? provider)
+            => Parse(s, provider, DateTimeStyles.None);
+
+        [RequiresPreviewFeatures]
+        static bool IParseable<DateOnly>.TryParse([NotNullWhen(true)] string? s, IFormatProvider? provider, out DateOnly result)
+            => TryParse(s!, provider, DateTimeStyles.None, out result);
```

Comment:
Why is the `!` needed here?

Reply:
Leftover from before I fixed https://github.com/dotnet/runtime/pull/53088

Reply:
Fixed.